src/winrtble/mod.rs

```diff
@@ -17,3 +17,17 @@ mod ble;
 pub mod manager;
 pub mod peripheral;
 pub mod utils;
+
+#[non_exhaustive]
+struct AdvertisementDataType;
```

Comment:
Why does this need to be a struct, rather than just a module?

Reply:
I don't remember, and looking back it it now, it makes more sense to be a module